modules/contentbox/models/exporters/ContentBoxExporter.cfc

```diff
@@ -47,8 +47,8 @@ component accessors=true {
 	property name="templateService" inject="id:emailtemplateService@contentbox";
 	property name="log" inject="logbox:logger:{this}";
 	property name="zipUtil" inject="zipUtil@contentbox";
-	property name="dataExporter" inject="id:dataExporter@cbadmin";
-	property name="fileExporter" inject="id:fileExporter@cbadmin";
+	property name="dataExporter" inject="id:dataExporter@contentbox";
```

Comment:
Actually, these are transients, not sure why they where injected. We can safely remove these from here please.